src/test/java/com/wikia/webdriver/testcases/auth/SignupTests.java

```diff
@@ -0,0 +1,284 @@
+    assertTrue(discussionPage.waitForPageReload().isStringInURL(PostsListPage.FULL_PATH),
+      "User should be redirected to discussion post list view upon sign up");
+  }
+
+  @Test(groups = MOBILE)
+  @Execute(onWikia = MercuryWikis.DISCUSSIONS_2)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userIsRedirectedToDiscussionPageUponSignUpFromDiscussionPageMobile() {
+    PostsListPage discussionPage = new PostsListPage().open();
+    signUpOnDiscussionMobilePageAs(discussionPage, createNewUser());
+    assertTrue(discussionPage.waitForPageReload().isStringInURL(PostsListPage.FULL_PATH),
+      "User should be redirected to discussion post list view upon sign up");
+  }
+
+  @Test(groups = DESKTOP)
+  public void passwordTogglerChangesPasswordVisibilityDesktop() {
+    performPasswordToggleTest(openSignUpModalOnDesktop());
+  }
+
+  @Test(groups = MOBILE)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void passwordTogglerChangesPasswordVisibilityMobile() {
+    performPasswordToggleTest(navigateToSignUpOnMobile());
+  }
+
+  /**
+   * HELPER METHODS
+   */
+
+  private SignUpUser createNewUser() {
+
+    return new SignUpUser(
+      String.format(USERNAME_PATTERN, Instant.now().getEpochSecond()),
+      getEmailAlias(userWithEmail.getEmail()),
+      String.format(PASS_PATTERN, Instant.now().getEpochSecond()),
+      BIRTH_DATE
+    );
+  }
+
+  private SignUpUser createUserWithExistingEmail() {
+    return new SignUpUser(
+      String.format(USERNAME_PATTERN, Instant.now().getEpochSecond()),
+      userWithEmail.getEmail(),
+      String.format(PASS_PATTERN, Instant.now().getEpochSecond()),
+      BIRTH_DATE
+    );
+  }
+
+  private SignUpUser createUserWithExistingUsername() {
+    return new SignUpUser(
+      existingUser.getUserName(),
+      getEmailAlias(userWithEmail.getEmail()),
+      String.format(PASS_PATTERN, Instant.now().getEpochSecond()),
+      BIRTH_DATE
+    );
+  }
+
+  private SignUpUser createUserWithPasswordMatchingUsername() {
+    String username = String.format(USERNAME_PATTERN, Instant.now().getEpochSecond());
+    return new SignUpUser(
+      username,
+      getEmailAlias(userWithEmail.getEmail()),
+      username,
+      BIRTH_DATE
+    );
+  }
+
+  private SignUpUser createTooYoungUser() {
+    return new SignUpUser(
+      String.format(USERNAME_PATTERN, Instant.now().getEpochSecond()),
+      getEmailAlias(userWithEmail.getEmail()),
+      String.format(PASS_PATTERN, Instant.now().getEpochSecond()),
+      LocalDate.now().minusYears(11)
+    );
+  }
+
+  private SignUpUser createNewUserWithSpecialCharacters() {
+    return new SignUpUser(
+      String.format("ユーザー%s", Instant.now().getEpochSecond()),
+      getEmailAlias(userWithEmail.getEmail()),
+      String.format("ユーザザー_%s", Instant.now().getEpochSecond()),
+      BIRTH_DATE
+    );
+  }
+
+  private String getEmailAlias(String email) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getEmailAlias" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
it is used